JSDemos/Demos/Charts/Strips/description.md

```diff
@@ -0,0 +1,5 @@
+A strip is a colored piece of the **Chart**'s background that highlights a range of values.
```

Comment:
> To configure strips, declare an array of objects in the argumentAxis.strips[] or valueAxis.strips[] property.

When reading for the first time, it was unclear why there are two options to add strips. Turned out that if `strips` is inside `argumentAxis`, the strip is vertical (horizontal for valueAxis). That's why the suggestion:
```suggestion
A strip is a colored piece of the **Chart**'s background that highlights a range of values on the argument or value axis.
```

Reply:
Sorry, but what are the other options except argument and value axes? We talk about a property that works (just like majority of other properties) for all possible axes.

Reply:
I don't like the explanation that "a strip is a piece of background". A strip is a regular word people should know. And if they don't, they'll figure it out anyway in this demo. 

How about something like this:

```suggestion
The **Chart** can display background strips that highlight value ranges.
```